### PR TITLE
Test cases for left mnrva 788

### PR DESCRIPTION
**Pull request by pras7430**
## JIRA:
See ticket here - https://jira.rax.io/browse/MNRVA-788

 ## Description:
Test cases added for dropdown loading and queryparams changes
 ## Testing:

 ## Screenshots:
(if applicable)

### CODE REVIEW

File: src/app/_features/visualize/pages/visualize.page/visualize.page.component.spec.ts [hunk 1]
```diff
@@ -7,6 +7,10 @@ import { ActivatedRoute } from '@angular/router';
 import { MetricsService } from 'src/app/_services/metrics/metrics.service';
 import { SharedModule } from '../../../../_shared/shared.module';
 import { envConfig, EnvironmentConfig } from 'src/app/_services/config/environmentConfig.service';
+import {
+  IMetric
+} from '../../../../_models/metrics';
```
Comment (FarajiA) [inline src/app/_features/visualize/pages/visualize.page/visualize.page.component.spec.ts:12]: Hey we have shorthand paths for this `@minerva/`

File: src/app/_features/visualize/pages/visualize.page/visualize.page.component.spec.ts [hunk 2]
```diff
@@ -17,13 +21,22 @@ const routes = [
...
+        start: '328833',
+        end: '8829938'
+      
     }
   }
 ];
 
+
+const metrics: IMetric[] = metricss;
+const metricDD=([].concat.apply([], metrics.filter(m => m.group==="network").map(a => a.metricName)) as any);
```
Comment (FarajiA) [inline src/app/_features/visualize/pages/visualize.page/visualize.page.component.spec.ts:37]: Fancy!

**Review by FarajiA — APPROVED** [review]

File: src/app/_features/visualize/pages/visualize.page/visualize.page.component.spec.ts (hunk 2 above)
Comment (pras7430) [reply src/app/_features/visualize/pages/visualize.page/visualize.page.component.spec.ts:37]: 😉

File: src/app/_features/visualize/pages/visualize.page/visualize.page.component.spec.ts (hunk 1 above)
Comment (pras7430) [reply src/app/_features/visualize/pages/visualize.page/visualize.page.component.spec.ts:12]: hmm

**Pull request merged**